lib/vmpooler/api/v1.rb

```diff
@@ -375,8 +390,16 @@ def atomically_allocate_vms(payload)
       content_type :json
       result = { 'ok' => false }
 
-      if jdata and !jdata.empty?
-        result = atomically_allocate_vms(jdata)
+      if jdata
+        empty = jdata.delete('empty')
+        invalid = jdata.delete('invalid')
+        statsd.increment(statsd_prefix + '.checkout.empty', empty) if !empty.nil?
```

Comment:
Because I just follow in whatever terrible footsteps came before me :)

Learning ruby by fixing up other non-idiomatic code - what could go wrong?  :trollface: